docs/NETWORK-MODELS.md

```diff
@@ -0,0 +1,41 @@
+# Network Models
+
+What are the network models that EVE supports for Edge Container workloads, and how are they implemented?
```

Comment:
I am not sure we need to keep repeating "Edge Container workloads" through this document. Why not stick to simply calling those "Edge Containers"?

Reply:
I am good either way. I felt I was under-using the term Edge Containers, so I started using them in a loose hybrid fashion. I will call them Edge Containers throughout.